### PR TITLE
Add Git prerequisite

### LINKED ISSUE #457: Document that `git` is a required tool
## Description

According https://github.com/boostercloud/booster/pull/456 issue, documentation should be improved to contain `git` as required tool on user computer.

### PR DESCRIPTION
**Pull request by jvmazagao**
## Description
Add Git prerequisite to documentation

## Changes
<Changes made> 

## Checks
- [ ] Project Builds
- [ ] Project passes tests and checks
- [x] Updated documentation accordingly
 
## Additional information

Fixes #457

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -201,6 +201,27 @@ versions:
   for Linux
 - [`nvm-windows`](https://github.com/coreybutler/nvm-windows) - Works with native Windows
 
+##### Install Git
+To initialize this project in your machine you need to install git
```
Comment (NickSeagull) [inline docs/README.md:205]: Perhaps something like:

```suggestion
Booster will initialize a Git repository when you create a new project, so it is required that you have it already installed it in your system.
```

File: docs/README.md [hunk 2]
```diff
@@ -201,6 +201,27 @@ versions:
...
+```shell
+brew install git
+```
+
+###### Windows
+
+```shell
+choco install git
+```
+
```
Comment (NickSeagull) [inline docs/README.md:224]: Could you add a comment here regarding that the user has the `user.email` and `user.name` properly configured globally?

**Review by NickSeagull — CHANGES_REQUESTED** [review]
Thanks a lot for this! I've left some suggestions

**Comment by jvmazagao** [comment]
finish the comments, can you take another look @NickSeagull

File: docs/README.md (hunk 2 above)
Comment (jvmazagao) [reply docs/README.md:224]: I make some changes

File: docs/README.md [hunk 3]
```diff
@@ -201,6 +201,30 @@ versions:
...
+After installing git in your machine, you need to take a look in the global variables `user.name` and `user.email`, that should be properly configured globally.
+you can can check this page to [config your global variables](https://git-scm.com/docs/git-config)
+
```
Comment (NickSeagull) [inline docs/README.md:227]: Awesome @jvmazagao , just a couple more suggestions

Can you write here the two steps to configure this?

Also can you please change `global variables` to `Git configuration variables`?

Comment (jvmazagao) [reply docs/README.md:227]: I will add the steps

File: docs/README.md [hunk 4]
```diff
@@ -201,6 +201,52 @@ versions:
...
+```
+
+###### Windows
+
+```shell
+choco install git
+```
+
+###### Git configuration variables
+After installing git in your machine, you need to take a look in the global variables `user.name` and `user.email`, that should be properly configured globally.
```
Comment (NickSeagull) [inline docs/README.md:226]:
```suggestion
After installing git in your machine, make sure that `user.name` and `user.email` are properly configured.
```

File: docs/README.md [hunk 5]
```diff
@@ -201,6 +201,52 @@ versions:
...
+you can can check this page for more informations: [Git configuration variables](https://git-scm.com/docs/git-config) and below you can configure the variables
```
Comment (NickSeagull) [inline docs/README.md:227]:
```suggestion
Take a look at the [Git configuration page](https://git-scm.com/docs/git-config) for more info.

To configure them, run in your terminal:
```

File: docs/README.md [hunk 6]
```diff
@@ -201,6 +201,52 @@ versions:
...
+```shell
+git config --global user.name "Your Name Here"
+git config --global user.email "your_email@youremail.com"
+```
+
+###### Windows
+
+```shell
+git config --global user.name "Your Name Here"
+git config --global user.email "your_email@youremail.com"
```
Comment (NickSeagull) [inline docs/README.md:247]: Those are all the same steps, so we can just leave them once:

```suggestion
```shell
git config --global user.name "Your Name Here"
git config --global user.email "your_email@youremail.com"
```
```

**Review by NickSeagull — CHANGES_REQUESTED** [review]
Looking good! Left last suggestions

**Review by NickSeagull — APPROVED** [review]
Awesome! Thanks

File: docs/README.md [hunk 7]
```diff
@@ -201,6 +201,38 @@ versions:
   for Linux
 - [`nvm-windows`](https://github.com/coreybutler/nvm-windows) - Works with native Windows
 
+##### Install Git
+Booster will initialize a Git repository when you create a new project, so it is required that you have it already installed it in your system.
```
Comment (alvaroloes) [inline docs/README.md:205]: What do you think about saying here that you can skip this with the corresponding flag? Suggestion:
```suggestion
Booster will initialize a Git repository when you create a new project (unless you use the `--skipGit` flag), so it is required that you have it already installed in your system.
```

(I also removed a duplicated "it" you had there 😉 )

**Review by alvaroloes — COMMENTED** [review]
Just one comment

**Review by javiertoledo — APPROVED** [review]

**Comment by jvmazagao** [comment]
Can we merge this?

**Comment by javiertoledo** [comment]
Done, thanks!

**Pull request merged**